bin/dev/vagrant-envrc

```diff
@@ -1,4 +1,10 @@
-ROOT=`readlink -f "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../"`
+
+# prevent cd from printing the directory it changes to. This breaks
+# cd/pwd constructions (See **).
+unset CDPATH
+
+# (**)
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)"
```

Comment:
$ROOT isn't needed any more, it was previously needed for `HCF_PACKAGE_COMPILATION_CACHE` in https://github.com/SUSE/scf/blob/46fbabad86c1a50c281f76462ef0f06e424a4536/bin/.runrc

runrc was renamed to vagrant-envrc